homeassistant/components/websocket_api/commands.py

```diff
@@ -178,7 +178,8 @@ def handle_get_states(hass, connection, msg):
             if entity_perm(state.entity_id, "read")
         ]
 
-    connection.send_message(messages.result_message(msg["id"], states))
+    cmd = connection.send_big_result if len(states) > 500 else connection.send_message
+    cmd(messages.result_message(msg["id"], states))
```

Comment:
`send_big_result` is a coroutine function, right? So we need to await it or schedule it.